installation: frontend assets install fails with pnpm 11

Package version (if known): latest 14.0.0b10.dev7
pnpm: v11.0.4


## Describe the bug
Running uv run invenio-cli install with `pnpm: v11.0.4` fails during frontend asset installation (related to this new feature: https://pnpm.io/cli/approve-builds). 
The generated asset workspace inherits `pnpm-workspace.yaml` from invenio_assets, but that file contains placeholder text under allowBuilds instead of valid boolean values.
```yaml
allowBuilds:
  '@fortawesome/fontawesome-free': set this to true or false
  '@parcel/watcher': set this to true or false
  '@swc/core': set this to true or false
````
As a result, pnpm does not approve required build scripts and aborts with `ERR_PNPM_IGNORED_BUILDS`.

## Steps to Reproduce
1. Use a clean InvenioRDM/latest instance setup.
2. Ensure the active pnpm is v11.x.
3. Run uv run invenio-cli install.
4. See error

## Expected behavior
should compile successfully

## Screenshots (if applicable)

```log
 ERR_PNPM_IGNORED_BUILDS  Ignored build scripts: @fortawesome/fontawesome-free@5.15.4, @parcel/watcher@2.5.6, @swc/core@1.15.33

Run "pnpm approve-builds" to pick which dependencies should be allowed to run scripts.
Traceback (most recent call last):
  File "/Users/INVENIO/invenio-dev-latest/.venv/bin/invenio", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 1514, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 1435, in main
    rv = self.invoke(ctx)
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 1902, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 1933, in invoke
    rv.append(sub_ctx.command.invoke(sub_ctx))
              ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 1298, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 853, in invoke
    return callback(*args, **kwargs)
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/decorators.py", line 34, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/flask/cli.py", line 400, in decorator
    return ctx.invoke(f, *args, **kwargs)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/click/core.py", line 853, in invoke
    return callback(*args, **kwargs)
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/flask_webpackext/cli.py", line 53, in install
    current_webpack.project.install(*args)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Users/INVENIO/invenio-dev-latest/.venv/lib/python3.14/site-packages/pywebpack/helpers.py", line 57, in inner
    raise RuntimeError("Process exited with code {}".format(exit_code))
RuntimeError: Process exited with code 1
Failed to install dependencies.

```

## Additional context

Replacing the placeholder with either `true` or `false` fixes the issue in https://github.com/inveniosoftware/invenio-assets/blob/master/invenio_assets/assets/pnpm-workspace.yaml
```yaml
allowBuilds:
  '@fortawesome/fontawesome-free': false
  '@parcel/watcher': false
  '@swc/core': true
```
```log
AI Reasoning on what to set true or false:

@swc/core should be true

In the generated asset workspace, @swc/core is a direct dev dependency used by the Rspack/SWC toolchain.
Its postinstall script validates that the native SWC binding is available and, if needed, attempts fallback handling.
Blocking this script increases the risk of platform-specific install/build failures, especially on environments where the native binding is not resolved cleanly.
Because this is part of the core frontend build toolchain, it is safer to explicitly allow it.
@parcel/watcher can be false

In this dependency graph, @parcel/watcher is only pulled in as an optional dependency of sass.
Its install script is used for source builds and is not required in the normal case where prebuilt platform packages are available.
This makes it a reasonable candidate to keep blocked by default.
@fortawesome/fontawesome-free can be false

In this dependency graph, it is pulled in transitively through replaywebpage.
Its post-install script only prints attribution text and does not appear to be required for frontend asset build functionality.
Blocking it should not affect normal InvenioRDM asset installation.
```

--- a/invenio_assets/assets/pnpm-workspace.yaml
+++ b/invenio_assets/assets/pnpm-workspace.yaml
@@ -1,3 +1,8 @@
+allowBuilds:
+  '@fortawesome/fontawesome-free': false
+  '@parcel/watcher': false
+  '@swc/core': false
+strictDepBuilds: false
 # required for pnpm to work with webpack
 # https://github.com/webpack/webpack/issues/5087#issuecomment-1589749984
 nodeLinker: hoisted

